packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -29,9 +30,10 @@ export type WithUTCOffsetType<T> = T & {
   attendees?: (Person & UTCOffset)[];
 };
 
-export type WebhookDataType = CalendarEvent &
-  EventTypeInfo & {
-    metadata?: { [key: string]: string | number | boolean | null };
+export type WebhookDataType = Partial<CalendarEvent> &
+  Partial<EventTypeInfo> & {
```

Comment:
Partial as open a new path for webhooks that can include different set of data and not just bookings